site/src/docs/getting-started/tutorials/figma-tutorial.mdx

```diff
@@ -0,0 +1,81 @@
+---
+slug: "/getting-started/tutorials/figma-basics"
```

Comment:
This `slug` should be `/getting-started/tutorials/figma-tutorial `because otherwise it tries to find `figma-basics.mdx`.